graphs/dijkstra_2.py

```diff
@@ -1,14 +1,41 @@
 def print_dist(dist, v):
+    """
+    Print vertex distances.
+
+    Parameters:
+    dist (list): A list of distances.
+    v (int): The number of vertices.
```

Comment:
Putting the type hints into the function signature allows [mypy](https://mypy.readthedocs.io) to check them in our tests.  Please do not repeat them in the docstring below because readers will get confused if one is changed but the other is not.
```suggestion
def print_dist(dist: list[float], v: int):
    """
    Print vertex distances.

    Parameters:
    dist: A list of distances.
    v: The number of vertices.
```